packages/wallet/test/integration/cip30mapping.test.ts

```diff
@@ -35,7 +35,7 @@ describe('cip30', () => {
     let mappedWallet: WalletApi;
 
     beforeAll(async () => {
-      mappedWallet = cip30.createWalletApi(wallet);
+      mappedWallet = cip30.createWalletApi(wallet, () => Promise.resolve(true));
```

Comment:
We also need case where user rejects the tx and verification that callback was called.